css/properties.json

```diff
@@ -120,6 +120,21 @@
     "order": "uniqueOrder",
     "status": "nonstandard"
   },
+  "-moz-context-properties": {
+    "syntax": "fill || stroke",
```

Comment:
Thanks for the input. I was confused about this. It can accept either, or both separated by a comma, so

fill
stroke
fill, stroke

Given that, is ```[ fill | stroke ]#``` still correct?